db/seeds.rb

```diff
@@ -1,7 +1,107 @@
+
+spag_21 = ArtistEvent.create!(artist: spag_west, event: cfac_2021)
+spag_22 = ArtistEvent.create!(artist: spag_west, event: cfac_2022)
+spag_ma = ArtistEvent.create!(artist: spag_west, event: mental_awakening)
+pg_21 = ArtistEvent.create!(artist: pluto_gang, event: cfac_2021)
+oc_22 = ArtistEvent.create!(artist: orange_constant, event: cfac_2022)
+oc_ma = ArtistEvent.create!(artist: orange_constant, event: mental_awakening)
+mgu_ma = ArtistEvent.create!(artist: mountain_grass_unit, event: mental_awakening)
+mgu_clt = ArtistEvent.create!(artist: mountain_grass_unit, event: cfac_clt)
+airshow_ma = ArtistEvent.create!(artist: airshow, event: mental_awakening)
+fixins_ma = ArtistEvent.create!(artist: fixins, event: mental_awakening)
+frute_clt = ArtistEvent.create!(artist: frute, event: cfac_clt)
+manic_22 = ArtistEvent.create!(artist: manic_vision, event: cfac_2022)
+
+inland = Sponsor.create!(name: "Inland Seafood", description: "", logo: "random logo")
+meraki = Sponsor.create!(name: "Meraki", description: "", logo: "random logo")
+ivester_jackson = Sponsor.create!(name: "Ivester Jackson", description: "", logo: "random logo")
+main = Sponsor.create!(name: "131 Main", description: "", logo: "random logo")
+southern_cut = Sponsor.create!(name: "Southern Cut Design and Build", description: "", logo: "random logo")
+vida = Sponsor.create!(name: "Vida Travel", description: "", logo: "random logo")
+lfg = Sponsor.create!(name: "Live Forever Golf", description: "", logo: "random logo")
+satisfied = Sponsor.create!(name: "Satisfied Entertainment", description: "", logo: "random logo")
+headynet = Sponsor.create!(name: "HeadyNet", description: "", logo: "random logo")
+kittle_homes = Sponsor.create!(name: "Kittle Homes", description: "", logo: "random logo")
+summit = Sponsor.create!(name: "Summit Investors", description: "", logo: "random logo")
+jnj = Sponsor.create!(name: "JNJ Apparel", description: "", logo: "random logo")
+prime = Sponsor.create!(name: "Prime Products", description: "", logo: "random logo")
+heath = Sponsor.create!(name: "Ricky Heath Plumbing", description: "", logo: "random logo")
+reliable = Sponsor.create!(name: "Reliable Restoration", description: "", logo: "random logo") 
+steady_hand = Sponsor.create!(name: "Steady Hand Brewery", description: "", logo: "random logo") 
+eargasm = Sponsor.create!(name: "Eargasm Earplugs", description: "", logo: "random logo")  
+barrys = Sponsor.create!(name: "Barry's", description: "", logo: "random logo") 
+supreme_lending = Sponsor.create!(name: "Supreme Lending", description: "", logo: "random logo") 
+suffolk_punch = Sponsor.create!(name: "Suffolk Punch Brewing", description: "", logo: "random logo")
+
+blake_house = Sponsor.create!(name: "The Blake House", description: "", logo: "random logo")
+will_to_live = Sponsor.create!(name: "Will to Live", description: "", logo: "random logo")
+youth_villages = Sponsor.create!(name: "Youth Villages", description: "", logo: "random logo")
```

Comment:
add description later

Reply:
I'm working with the directors on this as well. I need to follow up with my co-founder for the logos, too.